beer-garden/beer-garden
# #1063 - Stomp integration tests are failing

Closes #1063 

These changes address the failure of the stomp integration tests run by GitHub actions nightly. Items addressed:

* Renamed `stomp` test module to `stomp_` in order to avoid collision with the actual `stomp.py` package
* Changed base class of listener to match what Stomp expects
* Added property to listener, replacing class variable
* Added guard code to listener for the case where its method is called with a bad value
* Added needed decorator to test class
* Refactored ordering of test method so the logic is clearer
* Added files created by the *act* runner to `.gitignore`
* General code hygiene (comments refactored, imports cleaned up, dead code removed, type hints added)

## Test Instructions
* Sadly, because these changes seek to address the test runs on GitHub, the only *true* way to verify they are correct and satisfy the requirement is to wait for them to run on GitHub...
* That aside, the updated integration tests were verified to no longer fail when run on the local system and also on a simulated GitHub Actions environment via `act` (alas, with pinned versions of everything, so no matrix of versions).
